devenv.nix

```diff
@@ -189,7 +189,7 @@ in {
 
     # Environment variables
     env = lib.mkMerge [
-      (lib.mkIf cfg.enable {
+      (lib.mkIf (cfg.enable && !config.dotenv.enable) {
         DATABASE_URL = lib.mkDefault "mysql://shopware:shopware@127.0.0.1:3306/shopware";
         MAILER_URL = lib.mkDefault "smtp://127.0.0.1:1025?encryption=&auth_mode=";
         MAILER_DSN = lib.mkDefault "smtp://127.0.0.1:1025?encryption=&auth_mode=";
```

Comment:
We have to ensure, that the MAILER_URL/MAILER_DSN is **always** set to this to prevent any mail leakage.